npm/angular/package.json

```diff
@@ -59,6 +58,18 @@
   "publishConfig": {
     "access": "public"
   },
+  "nx": {
+    "targets": {
+      "build": {
+        "inputs": [
+          "{workspaceRoot}/scripts"
```

Comment:
We specify `/scripts` because we still rely upon many node scripts for handling building (amongst other things). Any changes to a script should invalidate this lib. Eventually we can become more targeted and stricter in the future but we want to be more conservative to start off